spyderlib/spyder.py

```diff
@@ -580,6 +580,11 @@ def setup(self):
                                             context=Qt.WidgetShortcut)
             self.register_shortcut(self.replace_action, "Editor",
                                    "Replace text")
+            self.file_switcher_action = create_action(self, _('File switcher...'), icon='filelist.png',
+                                            tip=_('Fast switch between files'), triggered=self.call_file_switcher,
+                                            context=Qt.ApplicationShortcut)

```

Comment:
Please make things fit in less than 79 columns for PEP8.

This is way too wide ;-)